app/helpers/project_builder.rb

```diff
@@ -0,0 +1,8 @@
+module ProjectBuilder
```

Comment:
Moreover the `helpers` folder in Rails has a meaning and is to use methods directly in the `views`. Which BTW is pattern I think should be use with caution too.